core/unit_test/sycl/TestSYCL_InterOp_Init.cpp

```diff
@@ -29,8 +29,7 @@ TEST(sycl, raw_sycl_interop) {
   Kokkos::Experimental::SYCL default_space;
   sycl::context default_context = default_space.sycl_queue().get_context();
 
-  sycl::default_selector device_selector;
-  sycl::queue queue(default_context, device_selector);
+  sycl::queue queue(default_context, sycl::default_selector_v);
```

Comment:
Presumably, this is just a new available syntax, not a deprecation.  Please propose these changes (prefer `sycl::default_selector_v `) in a separate PR.